extensions/wikia/PageHeader/PageHeaderController.class.php

```diff
@@ -2,9 +2,17 @@
 
 namespace Wikia\PageHeader;
 
+
 class PageHeaderController extends \WikiaController {
 
 	public function index() {
+		$title = \RequestContext::getMain()->getTitle();
+
+		$displayActionButton = !$title->isSpecialPage();// || $title->isSpecial( 'ThemeDesignerPreview' );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)